.github/workflows/build_test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           - { COMPILER: "gcc",   COMPILER_VERSION: "12", LINKER: "gold", CRYPTOLIB: "openssl" }
           - { COMPILER: "clang", COMPILER_VERSION: "12", LINKER: "gold", CRYPTOLIB: "gcrypt"  }
           - { COMPILER: "clang", COMPILER_VERSION: "13", LINKER: "lld",  CRYPTOLIB: "openssl" }
-          - { COMPILER: "clang", COMPILER_VERSION: "14", LINKER: "bfd",  CRYPTOLIB: "auto"    }
+          - { COMPILER: "clang", COMPILER_VERSION: "14", LINKER: "mold", CRYPTOLIB: "auto"    }
```

Comment:
Hmm, I think it'd be better to drop one of the `gold` tests instead. `gold` is on life-support, while `bfd` is very commonly used, so I think we should test both compilers with `bfd`.